crates/mod-mom/src/impls/endpoints/tenant/media.rs

```diff
@@ -38,8 +39,8 @@ async fn handle_ws(mut socket: ws::WebSocket, ts: Arc<MomTenantState>) {
         }
     }
 
-    // well if we can't close gracefully, we can't close gracefully.
-    _ = socket.close().await;
+    // FIXME: maybe implement graceful closing?
+    // See https://docs.rs/axum/latest/axum/extract/ws/enum.Message.html#variant.Close
```

Comment:
I don't know much about WebSockets. https://github.com/tokio-rs/axum/pull/2974 removed `.close()`, saying it's pretty useless.